datasets/hacker_news/pipelines/_images/run_csv_transform_kub/Dockerfile

```diff
@@ -10,25 +10,15 @@
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
+# limitations under the License.
 FROM python:3.8
-
-# Allow statements and log messages to appear in Cloud logs
 ENV PYTHONUNBUFFERED True
-
-# Copy the requirements file into the image
 COPY requirements.txt ./
-
-# Install the packages specified in the requirements file
 RUN python3 -m pip install --no-cache-dir -r requirements.txt
-
-# The WORKDIR instruction sets the working directory for any RUN, CMD,
-# ENTRYPOINT, COPY and ADD instructions that follow it in the Dockerfile.
-# If the WORKDIR doesn’t exist, it will be created even if it’s not used in
-# any subsequent Dockerfile instruction
+RUN python3 -m pip install fsspec
+RUN python3 -m pip install gcsfs
+RUN curl -sSL https://sdk.cloud.google.com | bash
+ENV PATH $PATH:/root/google-cloud-sdk/bin
 WORKDIR /custom
-
-# Copy the specific data processing script/s in the image under /custom/*
 COPY ./csv_transform.py .
-
-# Command to run the data processing script when the container is run
```

Comment:
Why are we removing these comments? Seem pretty useful for setting context to those unfamiliar with this code.

Reply:
Done.